tests/test_database.py

```diff
@@ -290,6 +298,21 @@ def test_write_session_associated_object_structure_created(example_database: Dat
     assert example_database.get_solutions_filename(example_subject.id, session.id).is_file()
     assert example_database.get_runs_filename(example_subject.id, session.id).is_file()
 
+def test_write_session_with_invalid_photoscan_id(example_database: Database, example_subject: Subject, example_transducer_tracking_result):
+    """ Test that when there is a transducer tracking result class associated with a session, the session
+    is correctly written to file."""
```

Comment:
Looks like this docstring got swapped with the docstring on the test just below it